### DISCUSSION TITLE
Change scrollbar side for items with overflow

### DISCUSSION THREAD

**Discussion by stackerito** [tailwindlabs/tailwindcss#5119] [Ideas]
I think it would be nice to be able to change the scrollbar from right to left without manually needing to write `ltr` and `rtl` CSS styles and possibly breaking the style

**Comment by simonswiss** [comment]
Hey!

You might be interested in this plugin:

https://www.npmjs.com/package/tailwindcss-rtl

**Reply by stackerito** [reply]
I saw it, but I wish it was some simple class in tailwind 😀
